### PR TITLE
docs: main README.md docker and other staff fixes

### PR DESCRIPTION
**Pull request by HiHat**
## Standards checklist:

- [ ] The PR title is descriptive.
- [ ] The PR doesn't replicate another PR which is already open.
- [ ] The PR has self-explained commits history.

- [ ] The code is mine or it's from somewhere with an Apache-2.0 compatible license.
- [ ] The code is efficient, to the best of my ability, and does not waste computer resources.
- [ ] The code is stable and I have tested it myself, to the best of my abilities.
- [ ] If the code introduces new classes, methods, I provide a valid use case for all of them.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

**Comment by andyoknen** [comment]
Great job! I left one small comment, otherwise I have no complaints.

File: README.md [hunk 1]
```diff
@@ -122,23 +169,22 @@ $ pocketcoind --help
...
-    $ rm -r ./checkpoints
     
     # unpack new checkpoint DB
     
     # for tar.gz archive
     $ tar -xzf latest.tgz -C ./
     
     # for bz2 archive
-    $ tar -xjf latest.tgz -C ./
+    $ tar -xjf latest.tbz -C ./
```
Comment (lostystyg) [inline README.md]: Its literally `tgz`, isn't it?

Comment (HiHat) [reply README.md]: 1. tbz = tar + bzip. Example description: https://docs.fileformat.com/compression/tbz/
2. There were no tbz snapshot provided actually, so this lines may be deleted

**Review by lostystyg — APPROVED** [review]

**Comment by lostystyg** [comment]
@HiHat  btw, it would be awesome if you could completely remove tbz related stuff because you were right - we are no longer providing tbz snapshots. At this point the PR will be completely solid.

**Comment by HiHat** [comment]
> @HiHat btw, it would be awesome if you could completely remove tbz related stuff because you were right - we are no longer providing tbz snapshots. At this point the PR will be completely solid.

Done!

**Review by andyoknen — APPROVED** [review]

**Pull request merged**